packages/base/field-component.gts

```diff
@@ -244,7 +244,7 @@ export function getBoxComponent(
       }
 
       .field-component-card.embedded-format {
-        padding: var(--boxel-sp);
+        padding: var(--boxel-field-embedded-padding, var(--boxel-sp));
```

Comment:
Im certain someone has comment on this. Essentially, embedded delegated render is offering a (non-optional) padding. I added this to remove the padding but this would also mean any nested embedded inside this embedded will also have 0 padding

Reply:
This is a known issue and it kicked off discussion of defining embedded format which is still WIP. I will just leave this code for now.